src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <section>
+      <form action="/search" method="get">
+        <label class="search-bar big" data-qa="big">
+          <img src="./images/Search.svg" alt="search" >
+          <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="input">
```

Comment:
Fix all cases for formatting
```suggestion
          <input
            type="text"
            placeholder="Try &quot;Los Angeles&quot;"
            class="input"
          >
```